installer/win/Symphony-x64.aip

```diff
@@ -43,23 +43,35 @@
     <ROW Directory="APPDIR" Directory_Parent="TARGETDIR" DefaultDir="APPDIR:." IsPseudoRoot="1"/>
     <ROW Directory="DesktopFolder" Directory_Parent="TARGETDIR" DefaultDir="DESKTO~1|DesktopFolder" IsPseudoRoot="1"/>
     <ROW Directory="ProgramMenuFolder" Directory_Parent="TARGETDIR" DefaultDir="PROGRA~1|ProgramMenuFolder" IsPseudoRoot="1"/>
+    <ROW Directory="Release_Dir" Directory_Parent="build_Dir" DefaultDir="Release"/>

```

Comment:
why are changes in .aip file necessary for this PR?

Reply:
because the spell checker module has native dependencies. Please see this doc -> https://perzoinc.atlassian.net/wiki/spaces/DES/pages/147161314/Electron+Spellchecker